doc/div-alg.xml

```diff
@@ -153,13 +122,16 @@ gap> W:=WedderburnDecompositionInfo(R);
 gap> SchurIndex(W[5]);
 3
 
-gap> G:=SmallGroup(40,1);              
+G:=SmallGroup(40,3);              
 <pc group of size 40 with 4 generators>
-gap> Size(Irr(G));                          
-16
-gap> SchurIndexByCharacter(GaussianRationals,G,16);
+gap> for i in [1..Length(Irr(G))] do 
+       if Size(KernelOfCharacter(Irr(G)[i]))=1 then 
```

Comment:
This will fail, because it does not have partial `> ` prompts. Same in several other examples below.